### PR TITLE
Enhance xcattest to support include in bundle file

### PR DESCRIPTION
**Pull request by penguhyang**
for example
rhels6.7_ppc64.bundle contains
```
...
#INCLUDE:rhels6.8_ppc64.bundle#
...
```

### CODE REVIEW

File: xCAT-test/xcattest [hunk 1]
```diff
@@ -731,6 +788,7 @@ sub loadcase
         log_this("Error: No case to run, please check the case name or command name");
     }
     log_this("Not to run:", @notrightcase) if (@notrightcase);
+    log_this("Duplicated case:", @duplicatedcase) if (@duplicatedcase);
```
Comment (hu-weihua) [inline xCAT-test/xcattest]: Could you handle ``#INCLUDEBAD:cannot open $file#`` at same time?

Comment (penguhyang) [inline xCAT-test/xcattest]: fixed.

File: xCAT-test/xcattest [hunk 2]
```diff
@@ -206,6 +208,57 @@ sub log_error
...
+    my $doneincludes = 0;
+    while (not $doneincludes) {
+        $doneincludes = 1;
+        if ($bundletext =~ /#INCLUDE:[^#^\n]+#/) {
+            $doneincludes = 0;
+            $bundletext =~ s/#INCLUDE:([^#^\n]+)#/include_file($1,$idir)/eg;
+        }
+    }
+    @caserange=split(",",$bundletext);
+    my %same;
```
Comment (penguhyang) [inline xCAT-test/xcattest]: It's used for identify whether this value is duplicated.

**Comment by hu-weihua** [comment]
I have reviewed and agree to merge. Thanks @penguhyang .

**Pull request merged**